code/game/turfs/closed/wall/reinf_walls.dm

```diff
@@ -19,6 +21,9 @@
 		pipe_astar_cost = 50 \
 	)
 
+/turf/closed/wall/r_wall/get_armour_list()
+	return list(MELEE = 30,  BULLET = 30, LASER = 20, ENERGY = 20, BOMB = 10, BIO = 100, RAD = 100, FIRE = 80, ACID = 70, STAMINA = 0)
```

Comment:
The reason we do it like this is because it saves on both memory and initialisation time due to not making redundant lists. There is proc call overhead but that only happens when a turf takes damage and since 80% of our turfs are space this will be **way** better.